### ISSUE TITLE
Use case (developer): support drag & drop map layers

### ISSUE THREAD

**Issue by AmeliaBR** [Maps4HTML/HTML-Map-Element-UseCases-Requirements#106]
The HTML map polyfill supports dragging mapML layers from URLs into a viewer & having them composite onto the displayed map.

This is probably not something we want to support by default — including arbitrary content from other websites on a viewer in my website.  But, the website author should be able to easily add such a functionality.

**Comment by prushforth** [comment]
I tend to agree. I implemented that as a way to demonstrate the power and simplicity of a map/layer DOM, but some maps aren't meant to be mashable, at least without explicit permission of the Web site owner. Could this fit into a block list that could be enabled (easily) by the Web site owner? It was not completely trivial to enable and test, and different browsers have a different attitude towards it, I found. Anyway, its mostly a show and tell feature, quite handy at this stage of developments.

**Comment by prushforth** [comment]
I think this may indirectly reveal a use case / capability that I can't find specifically in the document or in the issues here, maybe I just missed it.  This [issue](https://github.com/Maps4HTML/HTML-Map-Element-UseCases-Requirements/issues/41) may be related; the ability to bookmark something on the Web generally means it has a URL.

Specifically, the use case is to have an undifferentiated URL to a map layer or map document.  Take the example of a media type understood by Web browsers, but not part of HTML directly, e.g. image/png   When you paste the URL of an image into the URL bar of a browser, the browser receives metadata about the representation it is retrieving/ receiving, in the form of HTTP headers.  In the case of an image, once the browser satisfies itself it's receiving an image, it goes ahead and synthesizes an html document 'around' the image, allowing the regular parsing and rendering process to take place (as far as I understand the process at least).

In the case of a MapML document, it could follow the routine laid out by browsers for images, described above.  

But all of the above highlights a potential new use case: add map content to the browser by dereferencing a(n undifferentiated) URL.  If you all concur (let me know, somehow?) I will add an issue for the use case.
